docs/components/integration-framework/connectors/out-of-the-box-connectors/aws-sqs.md

```diff
@@ -0,0 +1,64 @@
+---
+id: aws-sqs
+title: AWS Simple Queue Service Connector
+description: Send messages to AWS Simple Queue Service from your BPMN process.
+---
+
+The AWS SQS Connector allows you to connect your BPMN service with the Amazon Web Service's [Simple Queue Service](https://aws.amazon.com/sqs/).
+
+## Prerequisites
+
+To use an **AWS SQS Connector**, you need to have an SQS Queue and IAM key and secret pair with `sqs:SendMessage` policy relative to your SQS. See the [SQS Developer Guide](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-basic-examples-of-iam-policies.html) to learn more.
```

Comment:
Have you thought about adding something like

```
... and configured as secret in your cluster. Follow [these steps in the appendix](...) to learn how to set this up.
```

+ small appendix section `Store SQS key and secret as secrets in your cluster`

==> Example https://docs.camunda.io/docs/next/components/integration-framework/connectors/out-of-the-box-connectors/slack/#appendix

Reply:
Done